### PR TITLE
Add 'Test if superseded' button to Batch and Queue pages in Portal

### PR DESCRIPTION
**Pull request by griffri**
Resolves https://github.com/dlcs/private-protagonist/issues/119

This PR adds the ability to test if a batch has been superseded in the queue and individual batch viewers, as seen in the previous version in the portal:

image
image

### CODE REVIEW

File: src/protagonist/Portal/Pages/Queue/Index.cshtml [hunk 1]
```diff
@@ -88,4 +88,33 @@ else
...
-@await Component.InvokeAsync("Pager", new { values = Model.PagerValues })
\ No newline at end of file
+@await Component.InvokeAsync("Pager", new { values = Model.PagerValues })
+
+@section Scripts 
+{
+    <script>
+         $('.test-button').click(function() {
+            let batchId = $(this).data('batch');
+            let button = this;
```
Comment (donaldgray) [inline src/protagonist/Portal/Pages/Queue/Index.cshtml:98]: (This isn't worth fixing, just an FYI) `let button = $(this);` should allow you use `button` without needing to cast as jquery element each time (`$(button).text('Superseded');` => `button.text('Superseded');`)

File: src/protagonist/Portal/Pages/Batches/Index.cshtml [hunk 2]
```diff
@@ -8,7 +8,7 @@
     var batchId = Model.Batch.Id.GetLastPathElement();
     var percentComplete = Math.Round(Model.Batch.Count > 0 ? (double)Model.Batch.Completed / (double)Model.Batch.Count * 100.0 : 1.0);
     var progressPct = Model.Batch.Finished.HasValue ? 100 : percentComplete;
-    ViewData["Title"] = $"Batch {batchId} - {Model.Batch.Count} images";
+    ViewData["Title"] = $"Batch {batchId} - {Model.Batch.Count} image{(Model.Batch.Count > 1 || Model.Batch.Count == 0 ? "s" : null)}";
```
Comment (donaldgray) [inline src/protagonist/Portal/Pages/Batches/Index.cshtml:11]: `(Model.Batch.Count > 1 || Model.Batch.Count == 0 ? "s" : null)` is difficult to understand at a glance, is `(Model.Batch.Count == 1 ? null : "s")` clearer?

**Review by donaldgray — APPROVED** [review]

File: src/protagonist/Portal/Pages/Queue/Index.cshtml (hunk 1 above)
Comment (griffri) [reply src/protagonist/Portal/Pages/Queue/Index.cshtml:98]: That's a good tip! Appreciated.

File: src/protagonist/Portal/Pages/Batches/Index.cshtml (hunk 2 above)
Comment (griffri) [reply src/protagonist/Portal/Pages/Batches/Index.cshtml:11]: Definitely! Will change this one, thanks for pointing it out.

**Pull request merged**